core/services/eth/contracts/FluxAggregator.go

```diff
@@ -75,7 +75,7 @@ func (fa *fluxAggregator) SubscribeToLogs(listener ethsvc.LogListener) (connecte
 type FluxAggregatorRoundState struct {
 	ReportableRoundID uint32   `abi:"_roundId"`
 	EligibleToSubmit  bool     `abi:"_eligibleToSubmit"`
-	LatestAnswer      *big.Int `abi:"_latestRoundAnswer"`
+	LatestAnswer      *big.Int `abi:"_latestResponse"`
```

Comment:
Isn't `latestRoundAnswer` more consistent with the other docs etc? Makes much more sense to me.

Reply:
In docs so far we've tried to make a distinction. Response is a single oracle's current view. Submission is the tuple of Round ID and Response. Answer is the aggregated value of many oracles.